src/helpers.cpp

```diff
@@ -678,6 +678,41 @@ std::string Helpers::toUpper(std::string const & s) {
     return lc;
 }
 
+// capitalizes one UTF-8 character in char array
+// works with Latin1 (1 byte) and Polish (2 bytes) characters
+// TODO add special characters that occur in other supported languages
+void Helpers::CharToUpperUTF8(char * c) {
+    switch (*c) {
+    case 0xC3:
+        if (*(c + 1) == 0xB3) //ó (0xC3,0xB3) -> Ó (0xC3,0x93)
+            *(c + 1) = 0x93;
```

Comment:
You can change this to match more special characters to:
```
        // grave, acute, circumflex, diaeresis, etc.
        if ((*(c + 1) >= 0xA0) && (*(c + 1) <= 0xBE)) {
            *(c + 1) -= 0x20;
        }
```

Reply:
when this is done I'll merge

Reply:
done. sorry for the delay

Reply:
thanks!

Reply:
ok to merge @MichaelDvP ?

Reply:
Yes.